src/MarkdownTextInput.web.tsx

```diff
@@ -156,6 +156,7 @@ const MarkdownTextInput = React.forwardRef<TextInput, MarkdownTextInputProps>(
     },
     ref,
   ) => {
+    const [isComposition, setIsComposition] = useState(false);
```

Comment:
Looks like we never read `isComposition` during React render and we probably also don't want to trigger a React render when this changes. How about using `useRef` here instead?